playbooks/roles/ocp-cronjob/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# defaults file for playbooks/roles/ocp-cronjob
+cronjob_setup_dir: ~/setup-files/cronjob-setup
+image: "busybox" 
+cronjob_namespace: "openshift-cronjob" 
+delete_cronjob: false # true or false
+job_name: "helloworld-cron" #can be update as per requirement
```

Comment:
nit: all these values can be updated as needed. The comment "can be update as per requirement" is redundant.